src/sage/numerical/interactive_simplex_method.py

```diff
@@ -3712,12 +3776,21 @@ def run_dual_simplex_method(self):
             output.append(self._html_())
         return HtmlFragment("\n".join(output))
 
-    def run_simplex_method(self):
+    def run_simplex_method(self, pivot_rule = None, *args):
         r"""
         Apply the simplex method and return all steps and intermediate states.
 
         If either entering or leaving variables were already set, they will be
         used.
+        
+                
+        INPUT:
+        
+        - :string: or :SimplexMethodPivot: Optional- set a pivot rule. 
+          Supported rules are "steepest_edge", "blands_rule", "dantzig", "NW_rule".
+          One can pass in a custom instance of `SimplexMethodPivot` too.
```

Comment:
Formatting for references to classes etc.: see https://deploy-preview-37575--sagemath.netlify.app/html/en/developer/sage_manuals#hyperlinks